packages/st2/component.makefile

```diff
@@ -7,18 +7,27 @@ ifneq (,$(wildcard /etc/debian_version))
 	DEBIAN := 1
 	DEB_DISTRO := $(shell lsb_release -cs)
 	DESTDIR ?= $(CURDIR)/debian/$(ST2_COMPONENT)
+else ifeq (,$(wildcard /etc/centos-release))
+   EL_DISTRO:=centos
+   EL_VERSION:=$(shell cat /etc/centos-release | awk 'NF>1{print $4}' | cut -d. -f1)
+else ifeq (,$(wildcard /etc/redhat-release))
+   EL_DISTRO:=redhat
+   EL_VERSION:=$(shell shell cat /etc/redhat-release | awk 'NF>1{print $4}' | cut -d. -f1)
 else
 	REDHAT := 1
 	DEB_DISTRO := unstable
 endif
 
-ifneq (,$(wildcard /usr/share/python/st2python/bin/python))
+ifeq ($(DEB_DISTRO),bionic)
+	PYTHON_BINARY := /usr/bin/python3
+	PIP_BINARY := /usr/bin/pip3
+else ifeq ($(EL_VERSION),8)
+    PYTHON_BINARY := /usr/bin/python3
+    PIP_BINARY := /usr/bin/pip3
```

Comment:
Makefile requires using `tabs` over `spaces`.
This needs to be fixed in other places as well.

Reply:
fixed

Reply:
I'm still seeing spaces inside Makefiles. Forgot to push the changes?

Reply:
re-fixed